docs/source/guide/shadows.md

```diff
@@ -0,0 +1,53 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Classical Shadows
+
+Classical shadows protocol aims to create an approximate classical representation
+of a quantum state using minimal measurements. The protocol is based on the idea of shadow tomography,
+which is a technique for reconstructing a quantum state from a small number of measurements.
+This approach not only characterizes and mitigates noise effectively but also retains
+sample efficiency and demonstrates noise resilience. For more details, see the section
+([Classical Shadow Protocol and its Robust Estimation](shadows-5-theory.md)).
+
+ 
+```{figure} ../img/shadows_workflow.png
+---
```

Comment:
If the figures are very close to published figures in some paper, please add a reference. If you instead designed them by yourself, no problem.

**This comment applies to all figures.**

Reply:
Thanks, all plots are my own, except the front picture of the classical_tutorial.md, but this one has been given a very long time ago and has been used on a lot of websites without the original authors given, I think it is alright for us to use that one.